### DISCUSSION TITLE
Auto Resize Permanently Stopped Working after Manually Setting Resolution in Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by MotorBottle** [utmapp/UTM#6225] [Linux]
**Describe the issue**
Auto Resize Permanently Stopped Working after Manually Setting Resolution in Ubuntu 20.04 Arm64 LTS. I manually set the resolution for some cases but then I found my self unable to re-enable the auto resize feature again.

There are 3 things noticed:
1. The problem accrued on version 4.4.5 and updating to 4.5.0beta did not fix it (VM was created in 4.4.5 and I don’t really know how to reinstall the spice tools for Linux, on Windows re-installing spice guest tools fixes this issue)
2. before logging in (VM just started), the resolution was correctly filling the screen but later it would change into the resolution set in settings and would not change any more even I rotate the screen.
3. I was able to manually reset the resolution to fit the screen by using command `sudo xrandr --output Virtual-1 --auto`, but it was a one-time fix, I need to use this command every time I change the scale/size of the window

**Configuration (required)**
* UTM Version: 4.4.5/4.5.0bets
* OS Version: 16.3.1
* Device Model: 11in 3rd Gen M1 8g
* VM: Ubuntu 20.04 Arm64 LTS (GPU Acceleration on, Angel - Metal)
* How did you install UTM? Misaka - TS2

**Comment by mark-pictor-csec** [comment]
I had a similar issue I fought with. I _thought_ I'd searched for it when initially encountering it, but apparently not - or didn't use the right keywords...

Anyway, I got it working again. I noticed, in one VM, that the refresh rate showed 75Hz. In the vm where resizing failed, the rate was 60Hz. I changed it to 75 and applied, and resizing the window started working again. 

After closing and re-opening the control panel, it shows 75Hz as the only option. I haven't restarted the VM since, but hopefully it keeps on working...

UTM version: 4.6.4
Host OS: 15.4.1
CPU: M4 Max
Guest OS: NixOS with KDE GUI
Accel: `virtio-ramfb-gl`
